test/hybrid_vpn_test.go

```diff
@@ -0,0 +1,235 @@
+package test
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/ssh"
+	"github.com/stretchr/testify/assert"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+)
+
+func TestTerraformGcpHybridVpnAws(t *testing.T) {
+	t.Parallel()
+	runId, _ := runID()
+	awsRegion := "ap-southeast-3"
+	awsAssumeRoleArn := "arn:aws:iam::106256755710:role/OrganizationAccountAccessRole"
+	gcpRegion := "asia-southeast2"
+	gcpProject := "test-terraform-project-01"
+
+	googleCredentials := getGoogleCredentials()
+
+	//
+	//
+	// AWS VPC
+	//
+	//
+
+	awsVpcBootstrapTerraformOptions := &terraform.Options{}
+	awsVpcDir := test_structure.CopyTerraformFolderToTemp(t, ".", "modules/terraform-aws-vpc/aws-vpc")
+
+	test_structure.RunTestStage(t, "create_aws_vpc_terraform_options", func() {
+		awsVpcInputs := map[string]interface{}{
+			"aws_region":          awsRegion,
+			"aws_assume_role_arn": awsAssumeRoleArn,
+			"name":                "vpc-" + runId,
+			"azs":                 []string{awsRegion + "a", awsRegion + "b", awsRegion + "c"},
+			"cidr":                "10.133.7.0/25",
+			"public_subnets":      []string{"10.133.7.64/26"},
+			"private_subnets":     []string{"10.133.7.0/26"},
+			"enable_flow_log":     false,
+			"flow_log_cloudwatch_log_group_retention_in_days": 0,
+		}
+
+		awsVpcBootstrapTerraformOptions = terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+			TerraformDir: awsVpcDir,
+			Vars:         awsVpcInputs,
+		})
+
+		_ = copySupportingFiles([]string{"provider-aws.tf"}, awsVpcDir)
```

Comment:
are we sure we don't want to handle the error and fail the test here? If an error is returned the test will fail at some point anyways, might as well fail as quickly as possible?

Reply:
Have added for this as there is no real reason, but kept it silent for deleting as that is quite useful incase you cleaned them up manually while a test is running (sad reason for a 30 min test to fail).